### PR TITLE
require LiipMonitorBundle and enable it in AppKernel

### PR DESCRIPTION
**Pull request by valiknet18**
ticket: https://app.asana.com/0/25002076043610/29172655422207

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -179,3 +179,68 @@ domain_routing:
...
+                path:                 /posts
+                status_code:          200
+                content:              null
+
+            posts_by_slug:
+                host:                 %api_domain%
+                port:                 80
+                path:                 /posts/duzhie-prosta-istoriia
+                status_code:          200
+                content:              null
```
Comment (spolischook) [inline app/config/config.yml]: empty line at the end of file needed

File: app/config/routing_api.yml [hunk 2]
```diff
@@ -17,3 +17,7 @@ performance-events:
 posts:
     type: rest
     resource: AppBundle\Controller\PostsController
+
+_monitor:
+    resource: "@LiipMonitorBundle/Resources/config/routing.xml"
+    prefix: /monitor/health
```
Comment (spolischook) [inline app/config/routing_api.yml]: empty line at the end of file needed

File: app/config/services.yml [hunk 3]
```diff
@@ -16,3 +16,12 @@ services:
...
             - { name: twig.extension }
+
+    monitor.check.php_extensions:
+        class: AppBundle\Check\TravisCheck
+        arguments:
+            - "geekhub-php"
+            - "CheTheatre"
+            - "AddPermisionForDashboard"
+        tags:
+            - { name: liip_monitor.check, alias: monitor_travis }
```
Comment (spolischook) [inline app/config/services.yml]: empty line at the end of file needed

File: src/AppBundle/Check/TravisCheck.php [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+class TravisCheck implements CheckInterface
+{
+    private $account;
+
+    private $repository;
+
+    private $branch;
+
+    public function __construct($account, $repository, $branch)
```
Comment (spolischook) [inline src/AppBundle/Check/TravisCheck.php]: doc block needed

File: src/AppBundle/Check/TravisCheck.php [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+namespace AppBundle\Check;
+
+use ZendDiagnostics\Check\CheckInterface;
+use GuzzleHttp\Client as Guzzle;
+use ZendDiagnostics\Result\Failure;
+use ZendDiagnostics\Result\Success;
+
+class TravisCheck implements CheckInterface
+{
+    private $account;
```
Comment (spolischook) [inline src/AppBundle/Check/TravisCheck.php]: doc block needed

File: src/AppBundle/Check/TravisCheck.php [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+    private $repository;
```
Comment (spolischook) [inline src/AppBundle/Check/TravisCheck.php]: doc block needed

File: src/AppBundle/Check/TravisCheck.php [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+    private $branch;
```
Comment (spolischook) [inline src/AppBundle/Check/TravisCheck.php]: doc block needed

File: src/AppBundle/Check/TravisCheck.php [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+    public function __construct($account, $repository, $branch)
+    {
+        $this->account = $account;
+        $this->repository = $repository;
+        $this->branch = $branch;
+    }
+
+    public function check()
```
Comment (spolischook) [inline src/AppBundle/Check/TravisCheck.php]: doc block needed

File: src/AppBundle/Check/TravisCheck.php [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+        }
+
+        return $result;
+    }
+
+    public function getLabel()
+    {
+        return 'Travis';
+    }
+}
```
Comment (spolischook) [inline src/AppBundle/Check/TravisCheck.php]: empty line at the end of file needed

**Comment by spolischook** [comment]
LGTM, fix code style and merge if it work

**Comment by spolischook** [comment]
use your fork and extend template

**Comment by mushkin-v** [comment]
LGTM

**Comment by iboved** [comment]
LGTM

**Pull request merged**